include/rocksdb/db.h

```diff
@@ -976,6 +976,9 @@ class DB {
     return IngestExternalFile(DefaultColumnFamily(), external_files, options);
   }
 
+  virtual Status VerifyChecksum() = 0;
+  virtual Status VerifyChecksum(const std::string& file_path) = 0;
```

Comment:
Why is it a function of DBImpl? Should it be a standalone function? Users should be able to call it without any DB opened. Ther emay not be any DB when they copy the SST file.